mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -1510,6 +1520,7 @@ func flattenComposerEnvironmentConfigPrivateEnvironmentConfig(envCfg *composer.P
 	}
 
 	transformed := make(map[string]interface{})
+	transformed["connection_type"] = envCfg.NetworkingConfig.ConnectionType
```

Comment:
Test is failing with a null reference exception. Seems like `NetworkingConfig` is sometimes null. Will need to guard for this. 
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x30e1a11]

goroutine 1785368 [running]:
github.com/hashicorp/terraform-provider-google-beta/google-beta/services/composer.flattenComposerEnvironmentConfigPrivateEnvironmentConfig(...)
	/go/src/github.com/hashicorp/terraform-provider-google-beta/google-beta/services/composer/resource_composer_environment.go:1494

```